services/motion/builtin/builtin_test.go

```diff
@@ -269,164 +266,6 @@ func TestMoveOnMapLongDistance(t *testing.T) {
 	test.That(t, spatialmath.PoseAlmostCoincidentEps(endPos.Pose(), goalInBaseFrame, 15), test.ShouldBeTrue)
 }
 
-func TestMoveOnMapPlans(t *testing.T) {
```

Comment:
Why are these tests being removed here?

Reply:
should be back now. they were originally removed since the destination was within the default plan dev mm which is 2.6 mm. at the time, there was no way to specify a motion cfg for a regular move on map call, but as of this morning we can!